### ISSUE TITLE
Display ACTUAL query time in aardvark's AQL editor #1112

### ISSUE THREAD

**Issue by aydn** [arangodb/arangodb#1770]
I think this solution ( https://github.com/arangodb/arangodb/commit/8cd57c3c68d3dae412403f5d0f823088eb4ba940 ) doesn't help us to "debug/benchmark/optimize AQL queries" ( https://github.com/arangodb/arangodb/issues/1112 ) 

We actually need to "`query exec. time`" 
 not "`query exec. time`" + "`client 2 server + server 2 client latency`" ( XHR - sending query / downloading results + JSON parsing + DNS lookup )

For example when I run:

```
var db = require('arangojs')();
db.useDatabase('test');

var aql = `
  FOR s IN smartphones
  LIMIT 10000
  
  RETURN {
    name: s.name,
    brand_name: DOCUMENT(s.brandId).brand_name,
    soc_name: DOCUMENT(s.socId).soc_name,
    os_name: DOCUMENT(s.osId).os_name
  }
`;

var start = Date.now();
db.query(aql, function(err, cursor){
	console.log(!!err, Date.now() - start);
});
```

Result is: `false 63`
 = 63ms

But the **Execution time: 0.532 s** in frontend
 = 532ms

**Comment by baslr** [comment]
My workaround is to set `slowQueryThreshold:0`.
And then find the right entry in the slow query log.

Enhancement would be to have the aql query id returned as a http header. `x-arango-query-id: 123`

**Comment by jsteemann** [comment]
The query execution time displayed in the web interface currently exaggerates the actual query execution time (ref: https://github.com/arangodb/arangodb/issues/1765). This will be fixed soon.
The workarounds until then is to measure the query execution time on the client-side or server-side (in a Foxx action).

**Comment by jsteemann** [comment]
Fixed in 2.8.5. The query execution time displayed is now the server-side query execution time, excluding HTTP transfer and excluding front-end sleep time.
